manuscript/react-modern/meet-the-react-component.md

```diff
@@ -49,18 +51,19 @@ function App() {
 
 export default App;
 # leanpub-end-insert
-~~~~~~~
+```
 
 Before we dive deeper into each topic, here comes a quick overview of what you are seeing:
 
-* First, this React component, called App component, is just a JavaScript function. In contrast to JavaScript functions, it's defined in [PascalCase](https://www.robinwieruch.de/javascript-naming-conventions). This kind of component is commonly called **function component**. Function components are the modern way of using components in React, however, be aware that there are other variations of React components too. (see **component types** later)
-* Second, the App component doesn't receive any parameters in its function signature yet. In the upcoming sections, you will learn how to pass information (props) from one component to another component. These props will be accessible via the function's signature as parameters then. (see **props** later)
-* And third, the App component returns code that resembles HTML. You will see how this new syntax, called JSX, allows you to combine JavaScript and HTML for displaying highly dynamic and interactive content in a browser. (see **JSX** later)
+- First, this React component, called App component, is just a JavaScript function. In contrast to JavaScript functions, it's defined in [PascalCase](https://www.robinwieruch.de/javascript-naming-conventions). This kind of component is commonly called **function component**. Function components are the modern way of using components in React, however, be aware that there are other variations of React components too. (see **component types** later)
```

Comment:
Suggestion: Add link to the **component types**, **props**, **JSX** for easy content navigation.